scripts/mafiachecker.js

```diff
@@ -22,6 +23,11 @@ function mafiaChecker() {
         theme.lowerCaseRoles = [];
         theme.correctSides = [];
         theme.lowerCaseSides = [];
+        globalStats = raw["stats"] || {};
+        
+        for (st in globalStats) {
```

Comment:
Creating global 'for' variable. Should be 'for (var st ...'.<br>The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.